crates/test-framework/Cargo.toml

```diff
@@ -9,25 +9,30 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
-app = { path = "../app" }
-spicepod = { path = "../spicepod" }
-tempfile = "3"
 anyhow = "1.0.95"
-serde_yaml.workspace = true
-clap.workspace = true
-reqwest.workspace = true
-tokio.workspace = true
-regex.workspace = true
+app = { path = "../app" }
 arrow = { workspace = true, features = ["prettyprint"] }
+async-openai = { workspace = true, optional = true }
+clap.workspace = true
 flight_client = { path = "../flight_client" }
 futures.workspace = true
-tonic.workspace = true
+indicatif = "0.17.9"
+rand = "0.8.4"
+regex.workspace = true
+reqwest.workspace = true
 rustls.workspace = true
-uuid = { workspace = true, features = ["v4"]}
+serde_json.workspace = true
+serde_yaml.workspace = true
+spicepod = { path = "../spicepod" }
 sysinfo = "0.33.1"
-indicatif = "0.17.9"
+tempfile = "3"
+tokio.workspace = true
+tonic.workspace = true
+uuid = { workspace = true, features = ["v4"] }
 
 [target.'cfg(not(windows))'.dependencies]
 nix = { version = "0.29", features = ["signal"] }
 
 [features]
+default = ["models"]
+models = ["dep:async-openai"]
```

Comment:
I don't think we need to feature flag the test operator - that just adds extra complexity for no real gain